examples/theano/robo_cartpole.py

```diff
@@ -0,0 +1,42 @@
+import gym, roboschool
+import numpy as np
+import random
+
+class NaivePolicy:
+    "Simple multi-layer perceptron policy, no internal state"
+    def __init__(self, observation_space, action_space):
+        return
+
+    def act(self, ob):
+        x = np.array([0.5-random.random()])
+        return x
+
+def demo_run():
+    env = gym.make("RoboschoolInvertedPendulum-v1")
+    pi = NaivePolicy(env.observation_space, env.action_space)
+
+    while 1:
```

Comment:
Because we are using Python 3.x, `True` is now a keyword and should be used in lieu of 1 (i.e. `while True:`)